VisualStudio.gitignore

```diff
@@ -224,6 +224,11 @@ ClientBin/
 *.publishsettings
 orleans.codegen.cs
 
+# Dump files containing OS information including username, network shares, etc. recorded during crash
```

Comment:
@rdev5 can we drop these changes from this template, now they have a more appropriate home?